File: Data Structures & Algorithms/maximum-subarray/submission-0.cpp
class Solution {
public:
    int maxSubArray(vector<int>& nums) {
         int n = nums.size() ;
         int count = 0;
         int mx = INT_MIN;

         for(int i=0;i<n;i++){
            int temp = nums[i] ;
            count += temp ;
            mx = max(count,mx) ;

            if(count <= 0){
                count = 0;
            }
         }

         return mx  ;
    }
};

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        n = len(nums) 
        num_set = set(nums) 
        maxsize = 0
        for it in num_set:
            if (it-1) not in num_set:
                count = 1
                currit = it
                while (currit+1) in num_set:
                    currit += 1
                    count += 1
                
                maxsize = max(maxsize,count) 

        return maxsize 



        